scripts/add_component_gate_trigger_job.sh

```diff
@@ -24,6 +24,7 @@ EOF
           --description "${issue_message}" \
           --label COMPONENT_GATE_TRIGGER \
           --label "${COMPONENT_NAME}" \
+          --label jenkins \
```

Comment:
Last line of the command shouldn't have `\` appended as the next line is not a continuation.

Reply:
Still shouldn't be there IMO.